source/common/router/header_parser.h

```diff
@@ -94,7 +96,7 @@ class HeaderParser : public Http::HeaderEvaluator {
                                              bool do_formatting = true) const;
 
   static std::string translateMetadataFormat(const std::string& header_value);
```

Comment:
Why translateMetadataFormat does not return `absl::StatusOr<T>`?

Reply:
I actually changed it to return StatusOr, but then saw that your original implementation intentionally returns the original header, and so I tried to keep it that way.
Alas, the use of `Match` and `StrReplaceAll` doesn't need `StatusOr` for both functions.

Reply:
Great!. Thanks.